### PR TITLE
Geojson2csv

### PR DESCRIPTION
**Pull request by smandaric**
Adds a Github Action to convert geoJSON files to CSV with WKT geometry column.

### CODE REVIEW

File: geojson2csv/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+RUN echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
+
+RUN apk add --no-cache \
+gcc \
+libc-dev \
+geos-dev \
+curl
+
+COPY requirements.txt /
```
Comment (judoole) [inline geojson2csv/Dockerfile]: These two line could be replaced by `pip install geojson shapely` I believe

File: geojson2csv/samples.geojson [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (judoole) [inline geojson2csv/samples.geojson]: I guess this file is only for local testing?

**Review by judoole — CHANGES_REQUESTED** [review]
Looks good!
We should add a README.md with usage.
Best would also be to add a Makefile with lint and test. But we could postpone that, and create an issue on it instead. That would also require us to use a special container for running the test/lint in Github Actions

File: geojson2csv/Dockerfile (hunk 1 above)
Comment (smandaric) [reply geojson2csv/Dockerfile]: Moved `pip install` to dockerfile

File: geojson2csv/samples.geojson (hunk 2 above)
Comment (smandaric) [reply geojson2csv/samples.geojson]: Yes, removing this for now. Easy to add when want to add testing.

**Review by judoole — APPROVED** [review]

**Pull request merged**